File: src/main/java/com/abaranya/rootseed/service/QuoteService.java
package com.abaranya.rootseed.service;

import com.abaranya.rootseed.entity.Quote;
import com.abaranya.rootseed.repository.QuoteRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import java.util.List;

public interface QuoteService {
    String getQuotes();
    Quote getLastQuote(String symbol);
    List <Quote> getDayQuote(String symbol);
}

File: src/main/resources/static/js/QuoteService.js
'use strict';

angular.module('rootseed.services', []).factory('QuoteService', ["$http", "CONSTANTS", function($http, CONSTANTS) {
    var service = {};
    service.getQuotes = function() {
        var url = CONSTANTS.getQuotes + userId;
        return $http.get(url);
    };
    service.getLastQuote = function() {
        return $http.get(CONSTANTS.getLastQuote);
    };
    return service;
}]);